app/tests/stdio_file_tests.c

```diff
@@ -0,0 +1,88 @@
+#ifdef WITH_LIB_FS
```

Comment:
Since this logically belongs to lib/fs can you move this over into lib/fs/test.c and if possible use the lib/unittest framework?